docs/devel-quick-start.md

```diff
@@ -64,7 +64,7 @@ That's all! You are ready to submit your first pull request!
 
 *Ramen* uses the `drenv` tool to build a development environment
 quickly. Please follow the
-[Setup instructions](https://github.com/RamenDR/ramen/tree/main/test#setup)
+[Setup instructions](../test/README.md#setup)
```

Comment:
Much better, the full links was always pain since when you preview a changed file it
sends you to the target on master instead of the version of the patch.